### DISCUSSION TITLE
co-authored by with bot user

### DISCUSSION THREAD

**Discussion by wyardley** [community/community#49727] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

So, we use bots (e.g., Renovate) to create PRs for things, in that case, dependency updates.

When squash-merging one of those PRs, I _think_ the Author should already be the bot user's username (you see only the bot user in the index when merging, unless you've also updated the branch)

And yet, when actually merging, GitHub always adds, e.g.,
`Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>`

to the body of the PR's commit message when squash merging. Since the merge already shows up having the (and in the examples I looked at, I don't even see `Committer` being set separately), why does it add a `Co-authored-by` note?

I usually edit it out, but this behavior seems sub-optimal to me.

**Comment by huxuan** [comment]
Same problem here, it also happens for self-hosted Renovate authenticated with a GitHub App token. The behavior now will be quite strange, is there any approach to avoid the automatically generated `Co-authored-by` note?

The following is a screenshot of the `git log`. As you can see, the `Author` is exactly the same as `Co-authored-by`.

![image](https://github.com/community/community/assets/726061/8e51e1d9-a0bd-46d5-a100-9e070719c0a3)
